packages/eth-json-rpc-provider/src/internal-provider.ts

```diff
@@ -103,17 +105,17 @@ export class InternalProvider {
    *
    * @param eip1193Request - The request to send.
    * @param callback - A function that is called upon the success or failure of the request.
-   * @deprecated Please use `request` instead.
+   * @deprecated Use {@link request} instead.
```

Comment:
I forgot we still had these methods around. If we are introducing breaking changes anyway, perhaps we should remove these methods in the upcoming release?

Reply:
Well, we really ought to stop using that library and use the provider directly. But true, maybe it's a larger effort to outright remove them.

Reply:
Would an issue for this be best in `/core` or some other location?